tests/integration/boolval_1.phpt

```diff
@@ -0,0 +1,15 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+
+v::boolVal()->validate(1);
+v::boolVal()->validate('on');
+v::boolVal()->validate('off');
+v::boolVal()->validate('yes');
+v::boolVal()->validate('no');
+v::boolVal()->validate(true);
+v::boolVal()->validate(false);
```

Comment:
I think that suppose to be `check()` and/or `assert()` methods, isn't? Otherwise we are just reproducing what was in the unit tests, right?